src/js/core.js

```diff
@@ -356,21 +356,20 @@ function MediumEditor(elements, options) {
         }
     }
 
-    function initPasteHandler() {
-        var pasteOptions = Util.extend(
-            {},
-            this.options.paste,
-            // Backwards compatability
-            {
-                forcePlainText: this.options.forcePlainText, // deprecated
-                cleanPastedHTML: this.options.cleanPastedHTML, // deprecated
-                disableReturn: this.options.disableReturn,
-                targetBlank: this.options.targetBlank,
-                contentWindow: this.options.contentWindow,
-                ownerDocument: this.options.ownerDocument
-        });
-
-        this.pasteHandler = new PasteHandler(this, pasteOptions);
+    function initPasteHandler(options) {
+        // Backwards compatability
+        var defaultsBC = {
+            forcePlainText: this.options.forcePlainText, // deprecated
```

Comment:
And I forgot, I actually already implemented this and we are already giving warnings when people use these props :stuck_out_tongue_winking_eye: 

the `mergeOptions()` method in `core.js` takes care of this.